Resources/Prototypes/Entities/Objects/Fun/dice.yml

```diff
@@ -9,12 +9,17 @@
   - type: Sprite
     sprite: Objects/Fun/dice.rsi
     noRot: true # If their sprites rotate, the number becomes even more illegible than usual.
+  - type: Tag
+    tags:
+    - Dice
+  - type: Item
+    size: 2
 
 - type: entity
   parent: BaseDice
   id: d100Dice
   name: d100
-  description: A die with one hundred sides! Probably not fairly weighted...
+  description: A die with ten sides. Works better for d100 rolls than a golf ball.
```

Comment:
But it's a d100?

Reply:
The actual d100 is like, a literal golf ball and not in yet, 
![image](https://user-images.githubusercontent.com/47410468/151923781-e086a599-a5be-47fc-863d-231089225ce5.png)
this is replacing the description with the one from TG for it.
This current 'd100' is basically the "d00" listed in TG, which is a 10 sided showing multiples of 10s, I wanted to rename it but the current visualizer determines the state names by its max side.